tests/foreman/ui/test_contentview.py

```diff
@@ -4111,3 +4111,33 @@ def test_positive_depsolve_with_module_errata(session, module_org):
             assert items['Name'] in rpm_pack
         assert result['module_streams']['table'][0]['Name'] == mod_stream
         assert result['errata']['table'][0]['Errata ID'] == FAKE_4_ERRATA_ID
+
+    @pytest.mark.stubbed
+    @pytest.mark.tier1
+    def test_positive_check_pulp3_is_installed():
+        """Ensure that pulp3 is installed
+
```

Comment:
Do we have anything in UI for pulp package installation verification?

Reply:
Services under Capsules provide Pulpcore version and supported content types.  That's the only means I know of to see that it's pulp3.

Reply:
I think we are already covering this in the CLI part so nothing extra needs to verify on the UI side right?